src/pages/Home.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PhoneBrand from '../components/PhoneBrand';
+import { getBrandsAction, filtreBrandsAction } from '../reducer/brands/brands';
+import Header from '../components/Header';
+
+const Home = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getBrandsAction());
+  }, [dispatch]);
+  const brandsData = useSelector((state) => state.brands);
+  const handlesearch = (param) => {
+    console.log(param);
+    dispatch(filtreBrandsAction(param));
+  };
+
+  return (<div>
+
+    <Header previewLink = "/" />
+    <div className='researchbar'>
+      <input type='text'
+      className='searchinput'
+      onChange={(e) => handlesearch(e.target.value)}
+      />
+      <button className='searchsubmit' type='button' onClick={handlesearch}>
+        Search
+      </button>
+    </div>
+    <ul className="brands-container">
+        {
+          brandsData.map((brandData) => (
+          <PhoneBrand
+          id={brandData.id}
+          key={brandData.id}
+          brand={brandData.name}
+          total={brandData.nbrePhone}
+          brandQuery={brandData.brandQuery}/>))
+        }
+
+    </ul>
+  </div>);
```

Comment:
[REQUIRED] Kindly ensure that your design follows the template as much as possible. For example in the template, there is no space between the elements and on the sides. But you have spaces there.
![image](https://user-images.githubusercontent.com/62938735/212144127-b52eb30d-8af8-4a58-88c7-30138c78f8f6.png)

[REQUIRED]You also don't have the top section present in the template.


![image](https://user-images.githubusercontent.com/62938735/212150170-03708c32-475d-4db6-abe7-825f14e0336d.png)

[REQUIRED] Kindly ensure that the texts and icons are positioned just like it is in the template